apps/jobs/models.py

```diff
@@ -3,37 +3,20 @@
 import datetime
 import logging
 
-from os.path import join
-
 from django.contrib.auth.models import User
 from django.db import models
 from django.db.models import Max
 from django.utils import timezone
 from rest_framework.exceptions import PermissionDenied
 
 from base.models import (TimeStampedModel, )
+from base.utils import RandomFileName
 from challenges.models import ChallengePhase
 from participants.models import ParticipantTeam
 
 logger = logging.getLogger(__name__)
 
 
-def submission_root(instance):
```

Comment:
I guess this will also involve updating the worker also. Concerned lines are 379 and 382